[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.application.HelloApplication  - Sorry, the main fxml could not be loaded
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.application.HelloApplication  - Sorry, the main fxml could not be loaded
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedjava.lang.NoClassDefFoundError: java/sql/SQLException
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.internal.util.config.ConfigurationException: Could not locate cfg.xml resource [hibernate.cfg.xml]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.application.HelloApplication  - Sorry, the main fxml could not be loaded
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.boot.registry.classloading.spi.ClassLoadingException: Unable to load class [bg.tu_varna.sit.library.data.entities.Task]
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedjava.lang.NullPointerException: Cannot invoke "org.hibernate.mapping.PersistentClass.getTable()" because "classMapping" is null
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedjava.lang.NullPointerException: Cannot invoke "org.hibernate.mapping.PersistentClass.getTable()" because "classMapping" is null
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedjava.lang.NullPointerException: Cannot invoke "org.hibernate.mapping.PersistentClass.getTable()" because "classMapping" is null
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - Get user error: org.hibernate.hql.internal.ast.QuerySyntaxException: USER is not mapped [SELECT t FROM USER t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.FORM references an unknown entity: bg.tu_varna.sit.library.data.entities.FORM
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.MappingException: Unable to find column with logical name: foreignidForm in org.hibernate.mapping.Table(FORM) and its related supertables and secondary tables
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Unknown mappedBy in: bg.tu_varna.sit.library.data.entities.USER.UserInfo, referenced property unknown: bg.tu_varna.sit.library.data.entities.UserInfo.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Unknown mappedBy in: bg.tu_varna.sit.library.data.entities.USER.UserInfo, referenced property unknown: bg.tu_varna.sit.library.data.entities.UserInfo.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Unknown mappedBy in: bg.tu_varna.sit.library.data.entities.UserType.byUSER, referenced property unknown: bg.tu_varna.sit.library.data.entities.USER.idUserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Illegal attempt to map a non collection as a @OneToMany, @ManyToMany or @CollectionOfElements: bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.idUserType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.userType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.userType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.userType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.status in bg.tu_varna.sit.library.data.entities.Status.byStatus
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Referenced property not a (One|Many)ToOne: bg.tu_varna.sit.library.data.entities.FORM.idForm in mappedBy of bg.tu_varna.sit.library.data.entities.FORM.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - Get user error: No row with the given identifier exists: [bg.tu_varna.sit.library.data.entities.UserType#0]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - Get user error: No row with the given identifier exists: [bg.tu_varna.sit.library.data.entities.UserType#0]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - Get user error: No row with the given identifier exists: [bg.tu_varna.sit.library.data.entities.UserType#0]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.USERTYEPE_idUserType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.UserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.USERTYEPE_idUserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.USERTYPE_idUserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.USERTYPE_idUserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.USERTYPE_idUserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.USERTYEPE_idUserType in bg.tu_varna.sit.library.data.entities.UserType.byType
